docs/how-to-restart-server.md

```diff
@@ -1 +1,12 @@
-# How do I restart my stardew server?
+# How do I restart my Junimohost Stardew server?
+
+We understand that occasional bugs can hinder your gameplay, but fret not – we've got a quick and easy solution for you. The best place to start when troubleshooting issues with your Junimohost server is to restart the server. This can be accomplished by following the below steps.
+
+What are the steps? (PC)
+1. Visit junimohost.com to access your account.
+2. Log into your Junimohost account using the 'Log In' button found in the top right hand corner of the homepage
+3. Once signed in, navigate to the left of the page and locate the 'Restart' button next to the 'Instructions' button, denoted in green.
+4. Click the 'Restart' button and allow the server to restart.
+
+If the issue persists, the best way to make us apparent to the issue is to reach out in our Discord server or alternatively contact us at contact@junimohost.com.
```

Comment:
same thing as the other file, this is not a good way to maintain up to date contact info. you can just link to the contact page for now.